components/locale-chooser.js

```diff
@@ -85,16 +88,31 @@ class LocaleChooser extends React.Component {
     open: false,
   }
 
-  handleClick() {
+  componentDidMount() {
+    document.addEventListener('click', this.clickOutside.bind(this), true)
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('click', this.clickOutside.bind(this), true)
```

Comment:
`this.clickOutside.bind(this)` returns an entirely new function, so this won’t actually remove the event listener added in `componentDidMount`.